tasks/CMakeLists.txt

```diff
@@ -8,7 +8,9 @@ add_dependencies(${ROCK_GAZEBO_TASKLIB_NAME}
 
 TARGET_LINK_LIBRARIES(${ROCK_GAZEBO_TASKLIB_NAME}
     ${OrocosRTT_LIBRARIES}
-    ${ROCK_GAZEBO_TASKLIB_DEPENDENT_LIBRARIES})
+    ${ROCK_GAZEBO_TASKLIB_DEPENDENT_LIBRARIES}
+    thruster_types)
```

Comment:
Which pkg-config package name did you use ? You were not generating any pkg-config file for the types, so you would have to use the pkg-config file for gazebo_thruster itself.

Anyways, this is what I meant: https://github.com/Brazilian-Institute-of-Robotics/simulation-gazebo_thruster/pull/2